Arrow/Sources/Arrow/ArrowType.swift

```
@@ -366,6 +417,24 @@ public class ArrowType {
             return ArrowTypeTime64(.microseconds)
         } else if  from == "ttn" {
             return ArrowTypeTime64(.nanoseconds)
+        } else if from.starts(with: "ts") {
+            let components = from.split(separator: ":", maxSplits: 1)
+            guard let unitPart = components.first, unitPart.count == 3 else {
+                throw ArrowError.notImplemented
+            }
+
+            let unitChar = unitPart.suffix(1)
+            let unit: ArrowTimestampUnit
+            switch unitChar {
+            case "s": unit = .seconds
+            case "m": unit = .milliseconds
+            case "u": unit = .microseconds
+            case "n": unit = .nanoseconds
+            default: unit = .milliseconds
```

Should we raise an exception here?
This case is for `tsX`, right?